README.md

```diff
@@ -479,6 +506,15 @@ be requested if the current time is later than the cached token expiry time minu
 | gid.connector.http.source.lookup.response.thread-pool.size    | optional | Sets the size of pool thread for HTTP lookup response processing. Increasing this value would mean that more concurrent requests can be processed in the same time. If not specified, the default value of 4 threads will be used.                                                                                                                                |
 | gid.connector.http.source.lookup.use-raw-authorization-header | optional | If set to `'true'`, uses the raw value set for the `Authorization` header, without transformation for Basic Authentication (base64, addition of "Basic " prefix). If not specified, defaults to `'false'`.                                                                                                                                                        |
 | gid.connector.http.source.lookup.request-callback             | optional | Specify which `HttpLookupPostRequestCallback` implementation to use. By default, it is set to `slf4j-lookup-logger` corresponding to `Slf4jHttpLookupPostRequestCallback`.                                                                                                                                                                                        |
+| gid.connector.http.source.lookup.connection.timeout           | optional | Source table connection timeout. |
+| gid.connector.http.source.lookup.success-codes                | optional | Comma separated http codes considered as success response. Use [1-5]XX for groups and '!' character for excluding. |
+| gid.connector.http.source.lookup.retry-codes                  | optional | Comma separated http codes considered as temporal errors. Use [1-5]XX for groups and '!' character for excluding.  |
+| gid.connector.http.source.lookup.ignored-response-codes       | optional | Comma separated http codes. Content for these responses will be ignored. Use [1-5]XX for groups and '!' character for excluding. Ignored response codes has to be a subset of `gid.connector.http.source.lookup.success-codes`.  |
+| gid.connector.http.source.lookup.retry-strategy.type          | optional | Auto retry strategy type: fixed_delay (default) or exponential_delay.  |
+| gid.connector.http.source.lookup.fixed-delay.delay            | optional | Fixed-delay interval between retries. Default 1 second.  |
```

Comment:
nit: I think it should be `gid.connector.http.source.lookup.retry-strategy.fixed-delay.delay`. This is how it is organized for Flink's restart strategy: https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/#fault-tolerance
The same for other retry strategy properties.